packages/cactus-api-client/package.json

```diff
@@ -64,6 +64,8 @@
     "@hyperledger/cactus-core": "1.0.0-rc.3",
     "@hyperledger/cactus-core-api": "1.0.0-rc.3",
     "@hyperledger/cactus-plugin-consortium-manual": "1.0.0-rc.3",
+    "@hyperledger/cactus-plugin-ledger-connector-besu": "1.0.0-rc.3",
```

Comment:
@outSH Please make it a separate package then so that we reduce the probability of running into circular dependencies in the future or just the bloat that you also mentioned.

Reply:
@petermetz Done